src/upstream_rbus.c

```diff
@@ -53,6 +61,25 @@ void subscribeRBUSevent()
         ParodusInfo("rbusEvent_Subscribe was successful\n");
 }
 
+#ifdef WAN_FAILOVER_SUPPORTED
+/* API to subscribe Active Interface name on value change event*/
+int subscribeCurrentActiveInterfaceEvent()
+{
+	int rc = RBUS_ERROR_SUCCESS;
+	ParodusPrint("Subscribing to Device.X_RDK_WanManager.CurrentActiveInterfaceEvent\n");
+	rc = rbusEvent_SubscribeAsync(rbus_Handle,WEBPA_INTERFACE,eventReceiveHandler,subscribeAsyncHandler,"parodusInterface",10*20);
+        if(rc != RBUS_ERROR_SUCCESS)
+	{
+		ParodusError("%s subscribe failed : %d\n", WEBPA_INTERFACE, rc);
+        }
+	else
+	{
+        	ParodusInfo("%s subscribe successful\n", WEBPA_INTERFACE);
```

Comment:
success log not required as subscribeAsyncHandler will indicate once it is success.